wasm/examples/nodejs/typescript/src/scriptBuilder.ts

```diff
@@ -0,0 +1,13 @@
+import { ScriptBuilder, Opcodes, addressFromScriptPublicKey, NetworkType } from "../../../../nodejs/kaspa"
+
+// An OpTrue is an always spendable script
+const myScript = new ScriptBuilder()
+                .addOp(Opcodes.OpTrue)
+
+const P2SHScript = myScript.createPayToScriptHashScript()
+const address = addressFromScriptPublicKey(P2SHScript, NetworkType.Mainnet)
+
+// Payable address
+console.log(address!.toString())
+// Unlock signature script
+console.log(myScript.createPayToScriptHashSignatureScript(""))
```

Comment:
Running `npx ts-node scriptBuilder.ts` runs into this error:

```
/Users/dev/.npm/_npx/1bf7c3c15bf47d04/node_modules/ts-node/src/index.ts:859
    return new TSError(diagnosticText, diagnosticCodes, diagnostics);
           ^
TSError: ⨯ Unable to compile TypeScript:
scriptBuilder.ts:13:22 - error TS2551: Property 'createPayToScriptHashSignatureScript' does not exist on type 'ScriptBuilder'. Did you mean 'encodePayToScriptHashSignatureScript'?

13 console.log(myScript.createPayToScriptHashSignatureScript(""))
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  ../../../../nodejs/kaspa/kaspa.d.ts:6832:3
    6832   encodePayToScriptHashSignatureScript(signature: HexString | Uint8Array): HexString;
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'encodePayToScriptHashSignatureScript' is declared here.
```

Reply:
Looks like an outdated example, must be replaced w encodePayToScriptHashSignatureScript

Reply:
Fixed this here: https://github.com/aspectron/rusty-kaspa/commit/98ba65271cd434f13b3cf98589331ddf5d703201

what carries a better meaning here... `encode` or `create`?

Reply:
technically its encoding a sigScript